src/resources/views/crud/fields/select2_from_ajax.blade.php

```diff
@@ -3,17 +3,33 @@
     $connected_entity = new $field['model'];
     $connected_entity_key_name = $connected_entity->getKeyName();
     $old_value = old(square_brackets_to_dots($field['name'])) ?? $field['value'] ?? $field['default'] ?? false;
+
+    $allows_null = $crud->model::isColumnNullable($field['name']) ?
+        ((isset($field['allows_null']) && $field['allows_null'] != false) || !isset($field['allows_null']) ? true : false) :
+        ((isset($field['allows_null']) && $field['allows_null'] != true) || !isset($field['allows_null']) ? false : true);
+
+    if ($old_value) {
+        $item = $connected_entity->find($old_value);
+
+    }else{
+        // In case the table column is not nullable, and developer didn't force it to be
+        // we select last inserted record in connected entity
+        // (Also works for nullable columns but developer forced allows_null => false)
+
+        if($allows_null == false) {
+           $item = $connected_entity::latest()->orderByDesc('id')->first();
```

Comment:
Possible issue: The connected entity might not have the "id" column. It might be called something else. To avoid this problem we can:
(A) Use ```$connected_entity_key_name``` defined above in this same file;
OR
(B) Not default to the last inserted item, but the first inserted item. So ```$connected_entity::first()```

(A) would probably be more helpful in real-life situations.
(B) would probably be more intuitive, because after you click the dropdown, you'd see all the options starting with the first one - so it makes perfect sense why that one was chosen as a default, because it's the first one;